internal/checker/terraform/rules/rancher/rancher2_cluster_v2_configuration.rego

```diff
@@ -0,0 +1,56 @@
+# METADATA
+# title: "Verify Rancher2 Cluster V2 Configuration"
+# description: "Policy ensures Rancher2 Cluster V2 configuration is correctly set up according to defined standards."
+# scope: package
+# related_resources:
+# - https://registry.terraform.io/providers/rancher/rancher2/latest/docs/resources/cluster_v2
+# custom:
+#   id: CB_TFRAN_076
+#   severity: MEDIUM
+package lib.terraform.CB_TFRAN_076
+
+import future.keywords.in
+
+isvalid(block) {
+	block.Type == "resource"
+	"rancher2_cluster_v2" in block.Labels
+}
+
+resource[resource] {
+	some block in pass
+	resource := concat(".", block.Labels)
+}
+
+resource[resource] {
+	some block in fail
+	resource := concat(".", block.Labels)
+}
+
+pass[block] {
+	some block in input
+	isvalid(block)
+	block.Attributes.name != ""
+    block.Attributes.kubernetes_version != ""
```

Comment:
![image](https://github.com/carbonetes/brainiac/assets/122254250/bfff3b48-32d6-4f5b-999d-dcd18faac07f)

Please ensure that the "kubernetes_version" field verifies whether its value matches one of the following values exclusively. ["rke2/k3s-version", "rke2-version", "k3s-version"]